DatadogCore/Tests/Datadog/RUM/RUMVitals/DisplayLinkerTests.swift

```diff
@@ -0,0 +1,103 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import XCTest
+@testable import DatadogRUM
+
+final class DisplayLinkerTests: XCTestCase {
```

Comment:
sanity-check/ These tests bring symptoms of flakiness - let's make sure it passes 1K reps in local 🙏 so we don't introduce issues in CI.